lib/src/components/molecules/star_rating.dart

```diff
@@ -117,7 +117,7 @@ class _ZdsStarRatingState extends State<ZdsStarRating> {
               duration: const Duration(milliseconds: 250),
               decoration: BoxDecoration(
                 color: colors.cool.shade30.withOpacity(_selected ? 1 : 0),
-                borderRadius: widget.rounded ? ZetaRadius.rounded : ZetaRadius.none,
+                borderRadius: widget.rounded ? BorderRadius.circular(8) : BorderRadius.circular(0),
```

Comment:
Should be `Zeta.of(context).radius.rounded : Zeta.of(context).radius.rounded.none`

Reply:
updated with Zeta